suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -996,6 +996,21 @@ export const selectDeviceModel = createMemoizedSelector(
     selectedDevice => selectedDevice?.features?.internal_model ?? null,
 );
 
+export const selectIsDeviceConnectedViaCable = createMemoizedSelector(
```

Comment:
this name seems wrong. Is not the selector rather saying that device is not connected via bluetooth? 
```suggestion
export const selectIsDeviceNotConnectedViaBluetooth = createMemoizedSelector(
```

Reply:
Removed and usage replaced with `selectIsDeviceConnectedViaBluetooth`.